### PR TITLE
Jkmarx update deprecated 1 8 items

### PR DESCRIPTION
**Pull request by jkmarx**
Ref #449 
Update template settings https://docs.djangoproject.com/en/1.11/ref/templates/upgrading/

### CODE REVIEW

File: refinery/config/settings/base.py [hunk 1]
```diff
@@ -138,23 +138,39 @@ def get_setting(name, settings=local_settings, default=None):
 # Make sure to set this to a random string in production
 SECRET_KEY = get_setting("SECRET_KEY")
 
-# List of callables that know how to import templates from various sources.
-TEMPLATE_LOADERS = (
-    'django.template.loaders.filesystem.Loader',
-    'django.template.loaders.app_directories.Loader',
-    # 'django.template.loaders.eggs.Loader',
-)
+TEMPLATES = [
```
Comment (jkmarx) [inline refinery/config/settings/base.py]: https://docs.djangoproject.com/en/1.11/ref/templates/upgrading/

File: refinery/config/settings/base.py [hunk 2]
```diff
@@ -138,23 +138,39 @@ def get_setting(name, settings=local_settings, default=None):
...
+                'django.contrib.messages.context_processors.messages',
+                'core.context_processors.extra_context',
+                'django.template.context_processors.request',
+                'django.contrib.auth.context_processors.auth',
+                'django.template.context_processors.debug',
+                'django.template.context_processors.i18n',
+                'django.template.context_processors.media',
+                'django.template.context_processors.static',
+                'django.template.context_processors.tz',
+                'django.contrib.messages.context_processors.messages',
```
Comment (mccalluc) [inline refinery/config/settings/base.py]: messages and auth are repeated?

Comment (jkmarx) [reply refinery/config/settings/base.py]: @mccalluc Sorry, thanks.

File: deployment/aws-config/config.yaml.template [hunk 3]
```diff
@@ -86,3 +86,6 @@ RDS_ROLE: "refinery"
 
 # The instance type for the EC2 instance
 # FIXED to "m3.medium"
+
+# Redirects all non-HTTPS requests to HTTPS, False is default.
+SECURE_SSL_REDIRECT = True
```
Comment (scottx611x) [inline deployment/aws-config/config.yaml.template]: Should remove this setting? Ref: https://github.com/refinery-platform/refinery-platform/pull/2586

Comment (jkmarx) [reply deployment/aws-config/config.yaml.template]: Yes. Branch switching mistake. Will remove.

**Pull request merged**